templates/README.3CRelease.md

```diff
@@ -0,0 +1,131 @@
+- **GitHub Release** - Verify that the Git tag and GitHub release were done
+- **Automated Pull Requests** - Review and merge PRs for Mender Docs, changelogs, etc
+- **Technical deliverables** - Verify the deliverables: Container images, binary packages, Yocto recipes
+- **Documentation updates** - Verify the formalities: Mender Docs updated, JIRA version released, changelogs published, license manifest published
+- **Communication** - Inform stakeholders and celebrate with pizza 🍕
+- **Follow-up** - Monitor and assist downstream repositories that consume the release
+
+## ASCII Flow Diagram
+
+```
+┌──────────────────┐
+│  CI/CD Pipeline  │
+│    Build, Test   │ ──┐
+│       ...        │   │
+│  (main branch)   │   │
+└──────────────────┘   │
+                       ▼
+              ┌────────────┐                        ┌────────────────┐
+              │ Release    │         Click,         │                │
+              │ Candidate  │ ──────> Click, ──────> │ Git tag +      │
+              │            │         Click!         │ GitHub Release │
+              └────────────┘                        └────┬───────────┘
+                                                         │
+                                                         ▼
+                                         ╔════════════════════════════════╗
+                                         ║ 🔵 Common Release Actions      ║
+                                         ║     (always executed)          ║
+                                         ╠════════════════════════════════╣
+                                         ║ ✓ Changelog                    ║──→ PR in mender-docs-changelog
+                                         ║   release-docs-changelog.yml   ║
+                                         ╟────────────────────────────────╢
+                                         ║ ✓ Mender Docs version          ║──→ PR in mender-docs (version update)
+                                         ║   release-mender-docs.yml      ║
+                                         ╟────────────────────────────────╢
+                                         ║ ✓ License manifest             ║──→ PR in mender-docs (license manifest update)
+                                         ║   release-oslicenses-golang.yml║
+                                         ║   TODO QA-1212                 ║
+                                         ║   release-oslicenses-generic.yml ║
+                                         ╟────────────────────────────────╢
+                                         ║ ✓ Atlassian Compass            ║──→ Deployment tracking updated
+                                         ║   release-compass.yml          ║
+                                         ╟────────────────────────────────╢
+                                         ║ ✗ JIRA version release         ║──→ All tickets mentioned ini the changelog marked with fixVersion
```

Comment:
I still think that those `X` may look confusing to someone who will try to figure out if this steep is not needed or it is not working.